C-Plus-Plus/cryptography/Affine_substitution_Cipher.cpp

```diff
@@ -0,0 +1,54 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+const int a = 19;
+const int b = 20;
+const int m = 26;
+
+string encryption(string message) {
+    string cipher;
+    for (int i = 0; i < message.size(); ++i) {
+        if(message[i]!=' ') {
+            cipher += (char)((a * (message[i] - 'A') + b) % 26 + 'A');
```

Comment:
@SangeetaMishra make a comment here i.e. ` to make it in range of ASCII alphabet addition of "A" is done`